dnf/base.py

```diff
@@ -2315,6 +2361,8 @@ def _merge_update_filters(self, q, pkg_spec=None, warning=True):
                     msg2 = _('No security updates needed for "{}", but {} '
                              'updates available').format(pkg_spec, count)
                     logger.warning(P_(msg1, msg2, count))
+            return merged_queries
```

Comment:
Why did you add this return? The final one on the next line should be enough.

Reply:
I will remove it.